Modules/Config/ConfigModule.lua

```diff
@@ -94,22 +94,69 @@ local options = {
             set = function()
                 ConfigModule.AceDB.profile.minimapIconEnabled = not ConfigModule.AceDB.profile.minimapIconEnabled
 
-                MinimapModule = BattlePetCompletionist:GetModule("MinimapModule")
+                local MinimapModule = BattlePetCompletionist:GetModule("MinimapModule")
                 MinimapModule:UpdateMinimap()
             end
         },
-        mapPinsHeader = {
+        brokerDescription = {
             order = 8,
+            name = "LibDataBroker settings" .. "\n",
```

Comment:
LibDataBroker makes sense to you and me, but I don't know if it is the best name for it, since people who doesn't have development experience wouldn't know what it is.
I however don't have a better idea at the moment - would "Data provider settings" be better?

Reply:
Maybe "Data Broker settings"?  That seems to match the name CurseForge uses for their category as well as how many of the Display addons refer to it. I could see an argument for just "Broker settings" as many people may not read any of the docs and just see the "Broker_" prefix to many of the plugin names and internalize that as what it's called.

Reply:
Lets go with `Data Broker settings` 🙂

Reply:
Change applied.